packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -87,15 +87,15 @@ $bpk-spacing-v2: true;
 ///   }
 
 @mixin bpk-input--valid {
-  padding-right: $bpk-spacing-base;
+  padding-right: bpk-spacing-xxl();
```

Comment:
Changed from 24px to 40px as the text was hidden under the icon. 32px could work too, but the space between text and icon would be very small